modules/@apostrophecms/ui/ui/apos/components/AposCellLabels.vue

```diff
@@ -30,6 +30,17 @@
         :modifiers="[ 'apos-is-filled', 'apos-is-danger' ]"
       />
     </span>
+    <span
+      v-for="({component, props, if: conditions}, index) in customCellIndicators"
+      :key="index"
```

Comment:
I would do `component` here instead of the index.

Reply:
What if we have multiple time the same component with different props like label? (Like it's done with `AposLabel`)
Not sure it would happen but might.

Reply:
Yes, makes sense. With different prosp. Ignore.